src/main/resources/db/postgres/schema.sql

```diff
@@ -1,102 +1,75 @@
+-- Create tables only if they don't exist
```

Comment:
**suggestion**: we could maybe refactor and modernize the PostgreSQL script following the work from the canonical version of Spring Petclinic: https://github.com/spring-projects/spring-petclinic/blob/main/src/main/resources/db/postgres/schema.sql and https://github.com/spring-projects/spring-petclinic/blob/main/src/main/resources/db/postgres/data.sql 
We should bear in mind that we support 3 DAO implementations

Reply:
Being closer to the canonical version certainly makes sense to me.